src/Endpoints/ShipmentEndpoint.php

```diff
@@ -45,4 +46,25 @@ public function create(Shipment $shipment): Shipment
 
         return $this->serializer->deserialize($data, Shipment::class);
     }
+
+    /**
+     * @param Shipment $shipment
+     * @return Matrix|null
+     * @throws AuthorizationFailedException
+     * @throws RequestFailedException
+     */
+    public function getMatrix(Shipment $shipment): ?Matrix
```

Comment:
waarom in het shipments endpoint? Is eigenlijk toch een eigen endpoint net als rates?

Reply:
in de API is het endpoint /shipments/matrix, daarom had ik hem in het ShipmentEndpoint gezet. Anderzijds heb je wel gelijk dat we naar de Rate ook een shipment object sturen die wel een eigen endpoint heeft.